h/app.py

```diff
@@ -94,6 +121,13 @@ def includeme(config):
         },
     })
 
+    # After commit events
+    config.add_request_method(EventQueue, name='after_commit_events', reify=True)
```

Comment:
As discussed in Slack, this is now implemented like the `h.features.Client` as a callable class. Adding events is done via `request.notify_after_commit(event)` as Sean suggested, but everything is done inside `h.app.EventQueue` (the internal `collections.deque`, adding the response callback, the response callback itself, etc.)